packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -335,6 +348,7 @@ const ButtonBase = React.forwardRef(function ButtonBase(inProps, ref) {
     focusRipple,
     tabIndex,
     focusVisible,
+    inclusiveDisabled,
```

Comment:
We might want to rebase on the unstyled button PR.

Reply:
That PR is not merged yet. Please keep the discussion focused.